NDTensors/src/blocksparse/linearalgebra.jl

```diff
@@ -200,7 +200,7 @@ function svd(
     if (sV * sVP) == -1
       Vb *= -1
     end
-    copyto!(expose(blockview(V, blockV)), expose(Vb))
+    copyto!((blockview(V, blockV)), (Vb))
```

Comment:
Why was this change made?

Also there are extraneous parentheses in your new code.

Reply:
Right I will remove the parenthesis. So here `blockview` and `Vb` are both `Tensors` so they call the `expose` version of `copyto!` which unexposes and calls the `tensor` version of `copyto` which calls the `storage` version of `copyto!` which calls the `exposed` version of `copyto` on the `data` of the storage. I imagine we would want this `copyto!` call to immediately move into the `tensor` version of the function (which it now does). I can revert if that would be better?

Reply:
The new code is fine as long as it works.